ql/instruments/bonds/amortizingfloatingratebond.hpp

```diff
@@ -34,13 +34,15 @@ namespace QuantLib {
     //! amortizing floating-rate bond (possibly capped and/or floored)
     class AmortizingFloatingRateBond : public Bond {
       public:
+        
         AmortizingFloatingRateBond(Natural settlementDays,
                                    const std::vector<Real>& notional,
                                    const Schedule& schedule,
                                    const ext::shared_ptr<IborIndex>& index,
                                    const DayCounter& accrualDayCounter,
                                    BusinessDayConvention paymentConvention = Following,
                                    Natural fixingDays = Null<Natural>(),
+                                   Natural paymentLag = Null<Natural>(),
```

Comment:
I'll move this to the end for backward compatibility.

Reply:
No issues.